config/blacklight.yml

```diff
@@ -1,6 +1,6 @@
 development:
   adapter: solr
-  url: <%= ENV['SOLR_URL'] || "http://127.0.0.1:#{ENV.fetch('SOLR_DEVELOPMENT_PORT', 8983)}/solr/hydra-development" %>
+  url: <%= ENV['SOLR_DEVELOPMENT_URL'] || "http://127.0.0.01:8983/solr/hydra-development" %>
```

Comment:
What's the advantage of doing `SOLR_DEVELOPMENT_URL` instead of using DOTENV to differentiate?

Reply:
Is `127.0.0.01` correct, or should this be `127.0.0.1`?

Reply:
I do. I think both are present in a vanilla Hyrax app, and if we're normalizing, I think we should be normalizing across environments.

DotEnv already allows us to set a `development` specific value in its own way, so hard coding this information into the variable name semantics seems fragile.